draft-kazuho-httpbis-priority.md

```diff
@@ -144,8 +143,13 @@ as chunks of the response arrive.
 
 The default value of the `progressive` parameter is `0`.
 
-A server that supports the Priority header field SHOULD distribute the bandwidth
-of a connection between progressive responses that share the same urgency.
+A server SHOULD distribute the bandwidth of a connection between progressive
+responses that share the same urgency.
+
+A server SHOULD transmit non-progressive responses one by one, in the order the
+requests were received, or if inferable, in the order the requests were
+generated.  Doing so maximizes the chance of the client making progress in
+processing the composition of the HTTP responses at the earliest moments.
```

Comment:
due to #14, maybe you want to say something different to "processing the composition"

Reply:
Good point.

So, should we merge #14? I think #14 to be an improvement. Then, we can change "processing" that appears here to "using" and merge this PR as well?

Reply:
As I write here, I notice #14 getting merged. Thank you! Making edits to this PR.